docs/upgrade-from-backup.md

```diff
@@ -0,0 +1,36 @@
+# Restore a 5.7 backup to an 8.0 cluster
+
+!!! important
+
+    During an upgrade or downgrade, you may encounter issues that Percona XtraBackup or Percona XtraDB Cluster are not designed to solve.
+
+For an upgrade or downgrade, use Percona XtraBackup to backup the source server data and restore the data to a target server, and then start a different version of Percona XtraDB Cluster on the restored data. 
```

Comment:
use Percona XtraBackup to backup -> back up

Reply:
There is way to downgrade. Only upgrade.

Reply:
Rewritten